examples/hubbard_su/main.jl

```diff
@@ -0,0 +1,121 @@
+using Markdown #hide
+md"""
+# Simple update for the Fermi-Hubbard model at half-filling
+
+Once again, we consider the Hubbard model but this time we obtain the ground-state PEPS by
+imaginary time evoluation. In particular, we'll use the [`SimpleUpdate`](@ref) algorithm.
+As a reminder, we define the Hubbard model as
+
+```math
+H = -t \sum_{\langle i,j \rangle} \sum_{\sigma} \left( c_{i,\sigma}^+ c_{j,\sigma}^- -
+c_{i,\sigma}^- c_{j,\sigma}^+ \right) + U \sum_i n_{i,\uparrow}n_{i,\downarrow} - \mu \sum_i n_i
+```
+
+with $\sigma \in \{\uparrow,\downarrow\}$ and $n_{i,\sigma} = c_{i,\sigma}^+ c_{i,\sigma}^-$.
+
+Let's get started by seeding the RNG and importing the required modules:
+"""
+
+using Random
+using TensorKit, PEPSKit
+Random.seed!(12329348592498);
+
+md"""
+## Defining the Hamiltonian
+
+First, we define the Hubbard model at $t=1$ hopping and $U=6$ using `Trivial` sectors for
+the particle and spin symmetries, and set $\mu = U/2$ for half-filling. The model will be
+constructed on a $2 \times 2$ unit cell, so we have:
+"""
+
+t = 1
+U = 6
+Nr, Nc = 2, 2
+H = hubbard_model(Float64, Trivial, Trivial, InfiniteSquare(Nr, Nc); t, U, mu=U / 2);
+
+md"""
+## Running the simple update algorithm
+
+Next, we'll specify the virtual PEPS bond dimension and define the fermionic physical and
+virtual spaces. The simple update algorithm evolves an infinite PEPS with weights on the
+virtual bonds, so we here need to intialize an [`InfiniteWeightPEPS`](@ref). By default,
+the bond weights will be identity. Unlike in the other examples, we here use tensors with
+real `Float64` entries:
+"""
+
+Dbond = 8
+physical_space = Vect[fℤ₂](0 => 2, 1 => 2)
+virtual_space = Vect[fℤ₂](0 => Dbond / 2, 1 => Dbond / 2)
+wpeps = InfiniteWeightPEPS(rand, Float64, physical_space, virtual_space; unitcell=(Nr, Nc));
+
+md"""
+Before starting the simple update routine, we normalize the vertex tensors of `wpeps` by
+dividing with the maximal vertex element (using the infinity norm):
+"""
+
+for ind in CartesianIndices(wpeps.vertices)
+    wpeps.vertices[ind] /= norm(wpeps.vertices[ind], Inf)
+end
```

Comment:
```suggestion
normalize!.(wpeps.vertices, Inf)
```
But is this necessary? since these are initialized as identity, isn't the infinity norm just `1`?

Reply:
Didn't think about this but you're right: this doesn't change anything so I just removed the normalization here